tools/webpack-plugin/tests/e2e/createE2ETest.ts

```diff
@@ -19,9 +19,16 @@ export function createE2ETest(configBuilder: (mode: webpack.Configuration['mode'
         }
 
         function mockProcessor() {
-            return jest
-                .spyOn(SourceProcessor.prototype, 'processSourceAndSourceMapFiles')
-                .mockImplementation(async (_, __, debugId) => Ok({ debugId: debugId ?? 'debugId' } as never));
+            return jest.spyOn(SourceProcessor.prototype, 'processSourceAndSourceMapFiles');
+            // .mockImplementation(async (sourcePath, sourceMapPath, debugId) =>
```

Comment:
1. dead code - can we remove it?
2. Why are you changing it?

Reply:
1. yes
2. there's no need to mock this